libcxx/include/__atomic/atomic_ref.h

```diff
@@ -95,10 +95,14 @@ struct __atomic_ref_base {
 
   friend struct __atomic_waitable_traits<__atomic_ref_base<_Tp>>;
 
+  // require types that are 1, 2, 4, 8, or 16 bytes in length to be aligned to at least their size to allow them to be
```

Comment:
I'd say "to be potentially used lock-free". We don't know if they _actually_ can or not on a given target.

Reply:
Done.  Thanks.